aspnet/web-forms/overview/older-versions-security/introduction/forms-authentication-configuration-and-advanced-topics-cs.md

```diff
@@ -197,7 +197,7 @@ By default, the keys used for encryption and validation are generated automatica
 
 When working in a web farm setting or sharing authentication tickets across applications on the same server, you will need to configure the &lt;machineKey&gt; element in the affected applications so that their decryptionKey and validationKey values match up.
```

Comment:
https://docs.microsoft.com/en-us/aspnet/web-forms/overview/older-versions-security/introduction/forms-authentication-configuration-and-advanced-topics-cs

This also has too many passwords to salvage. I'm afraid this one needs to be deleted too.
Please review all the doc's that are flagged and search for "password". If they have more than 2 occurrences, let me know.

Reply:
-vb version has the same issues.  Will remove that one was well.